assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -508,6 +512,16 @@ export const selectors = {
 	 * @return {(boolean|undefined)} `true` if the Web Stories plugin is enabled, `false` if not. Returns `undefined` if not loaded.
 	 */
 	isWebStoriesActive: getSiteInfoProperty( 'webStoriesActive' ),
+
+	/**
+	 * Returns true if this site has multiple users.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean|undefined)} `true` if the site has multiple users, `false` if not. Returns `undefined` if not loaded.
+	 */
+	isMultipleUsers: getSiteInfoProperty( 'multipleUsers' ),
```

Comment:
The name should be updated to reflect the scope of the boolean which as previously mentioned should be whether or not the site has multiple users who can use Site Kit. With this in mind, I think perhaps `hasMultipleCapableUsers` would be more apt.

Reply:
Agreed, see above; let's call this `hasMultipleAdmins`.